data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/model/ExportSummary.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.dynamodb.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Model represents the summary manifest information of the export.
+ * <p>
+ * Check more details from
+ * https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/S3DataExport.Output.html#S3DataExport.Output_Manifest
+ * </p>
+ */
+public class ExportSummary {
```

Comment:
We should probably have a `@JsonIgnoreProperties(ignoreUnknown = true)` annotation on this. This way if Dynamo starts adding new fields, this code does not break.